### PR TITLE
Add random bool processor

### PR DESCRIPTION
**Pull request by marlontrapp**
Adding an random boolean processor.

### CODE REVIEW

File: processors.go [hunk 1]
```diff
@@ -198,6 +199,15 @@ func ProcessorCompanyName(cmap *ColumnMapper, input string) (string, error) {
 	return fake.Company(), nil
 }
 
+// ProcessorRandomBoolean will return a random boolean value.
+func ProcessorRandomBoolean(cmap *ColumnMapper, input string) (string, error) {
```
Comment (junkert) [inline processors.go]: What are your thoughts about renaming this function to ProcessorRandomBooleanString? My concern here is that function is returning a string representation of a boolean and not an actual Go bool type.  If we end up adding a ProcessorRandomBoolean in the future that returns a bool things may get a little confusing.

Comment (junkert) [reply processors.go]: Bah I see what you are doing now. Ignore my above comments.

**Comment by marlontrapp** [comment]
@junkert Is there anything else you'd like to change on that? Or could we merge this one?

**Comment by junkert** [comment]
It looks like I am unable to rerun your test. I'm going to push this through and see if testing passes. It looks like there was an error pulling Goveralls to the project which may have been a blip causing the tests to fail. @wavemoran is there a security setting to allow me to re-run PR tests that fail?

Screen Shot 2020-02-06 at 1 08 06 PM

**Comment by marlontrapp** [comment]
I think it was some problem with mine CircleCI. I think it worked in master right? Thank you!

**Comment by wavemoran** [comment]
@junkert that looks like @marlontrapp's CircleCI as he mentioned. Everything looks green for master.

**Comment by junkert** [comment]
Yep looks ok. Thanks everyone.

**Pull request merged**